xml/System.Net.Sockets/Socket.xml

```diff
@@ -8341,8 +8341,8 @@ The GetHashCode method returns a hash code of this instance. This value can be u
         <ReturnType>System.Net.Sockets.SafeSocketHandle</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a <see cref="T:System.Net.Sockets.SafeSocketHandle" /> that represents the socket handle that the current <see cref="T:System.Net.Sockets.Socket" /> object encapsulates.</summary>
+        <value>A socket handle exposed in a safe manner for the socket that the current <see cref="T:System.Net.Sockets.Socket" /> object encapsulates.</value>
         <remarks>To be added.</remarks>
```

Comment:
Lets add something warning how using the safe handle to operate on the socket directly may result in an undocumented state. We just hit this in https://github.com/dotnet/corefx/issues/39677.

Reply:
This would take the following form:
```suggestion
        <remarks>  
          <format type="text/markdown"><![CDATA[  

> [!IMPORTANT]
> Using the <xref:System.Net.Sockets.SafeSocketHandle> to operate on the socket directly may result in an undocumented state. 

         ]]></format>
         </remarks>
```